static/gsApp/views/amCheckout/steps/reviewAndConfirm.tsx

```diff
@@ -222,19 +222,18 @@ function ReviewAndConfirmHeader({
   const effectiveNow = new Date(effectiveAt).getTime() <= new Date().getTime() + 3600;
 
   let subText;
-  if (effectiveNow) {
+  if (subscription.isSelfServePartner) {
     subText = tct(
-      'These changes will apply immediately, and you will be billed today[partnerName].',
+      'These changes will apply [applyDate], and you will be billed by [partnerName] monthly for any recurring subscription fees and incurred pay-as-you-go fees.',
```

Comment:
probably doesn't matter since i doubt we'll have any self serve partners on older plans, but if we do we'll need to special case "pay-as-you-go" as well